ansible_base/authentication/views/ui_auth.py

```diff
@@ -25,9 +25,9 @@ def get(self, request, format=None):
 def generate_ui_auth_data():
     authenticators = Authenticator.objects.filter(enabled=True)
     response = {
-        'show_login_form': False,
         'passwords': [],
         'ssos': [],
+        'show_login_form': False,
```

Comment:
We are not going to nix this one? If not, can we move it back up to prevent the "random" change.

Reply:
Yeah, apparently it controls if the login buttons appear.